spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -1308,3 +1308,87 @@ for (Pageable p = Pageable.ofSize(25).sortBy(Sort.desc("yearBorn"),
   p = page.nextPageable();
 }
 ----
+
+===== Scenario: Person Entity and People Repository
+
+This keyset cursor-based pagination scenario uses the same `Person` entity and example dataset from the offset-based pagination scenario, but orders it by `name` and then by `id`,
+
+[source,json]
+----
+[
+   {"id":3, "name":"Alyse Dadson"},
+   {"id":6, "name":"Boothe Martinson"},
+   {"id":10, "name":"Boothe Martinson"},
+   {"id":2, "name":"Corri Davidou"},
+   {"id":8, "name":"Danita Pilipyak"},
+   {"id":9, "name":"Harlene Branigan"},
+   {"id":5, "name":"Jaquith Wealthall"},
+   {"id":1, "name":"Lin Le Marchant"},
+   {"id":4, "name":"Orelle Roughey"},
+   {"id":7, "name":"Patten Bedell"}
+]
+----
+
+[source,java]
+----
+@Repository
+public interface People extends BasicRepository<Person, Long> {
+    KeysetAwarePage<Person> findAll(Pageable pagination);
+}
+----
+
+Code Execution:
+
+[source,java]
+----
+@Inject
+People people;
+
+Pageable firstPageRequest = Pageable.ofSize(4).sortBy(Sort.asc("name"), Sort.asc("id"));
```

Comment:
This will generate a Pageable of offset, is not it?

Reply:
Yes, it requests the first keyset aware page with an offset from the beginning of the data set.  Then, once it has that page, it requests the next page, which is via a cursor based on the keyset of the last element of the first page.  This is typical usage if you wanted to start at the beginning and read through all of the results making sure you don't miss any due to deletions of other entities.